packages/drivers/redux-form/src/selectors.js

```diff
@@ -0,0 +1,42 @@
+import * as reduxform from 'redux-form'
+
+// selectors of redux-form
+export const selectorNames = [
+  'getFormValues',
+  'getFormInitialValues',
+  'getFormSyncErrors',
+  'getFormMeta',
+  'getFormAsyncErrors',
+  'getFormSyncWarnings',
+  'getFormSubmitErrors',
+  'getFormError',
+  'isDirty',
+  'isPristine',
+  'isValid',
+  'isInvalid',
+  'isSubmitting',
+  'hasSubmitSucceeded',
+  'hasSubmitFailed',
+]
+
+// selectors without parameters
+export const selectorNamesWithoutParameter = [
```

Comment:
`WithoutNameParameter` ?

Reply:
i think not, this is the names of selectors whitout parameter.